bindings/c/test/mako/mako.cpp

```diff
@@ -458,10 +458,15 @@ int populate(Database db,
 	return 0;
 }
 
-void updateErrorStatsRunMode(ThreadStatistics& stats, fdb::Error err, int op) {
+void updateErrorStatsRunMode(ThreadStatistics& stats, const Arguments& args, fdb::Error err, int op) {
 	if (err) {
 		if (err.is(1020 /*not_commited*/)) {
 			stats.incrConflictCount();
+		} else if (err.is(1031 /*timeout*/)) {
+			if (args.transaction_timeout == 0) {
```

Comment:
I think if we have timeouts as a separate category for statitistics, we shouldn't count them again as errors. That could introduce a confusion.

Reply:
I will change this. Note that this change might require changes to `printStats` later.